types/Artist.tsx

```diff
@@ -8,18 +7,48 @@ export type Artist = {
   popularity: number;
 };
 
-export type ArtistRecord = {
+export type SOCIAL = {
+  id: string;
+  bio: string | null;
+  avatar: string | null;
+  region: string | null;
+  username: string | null;
+  updated_at: string;
+  profile_url: string;
+  followerCount: number;
+  followingCount: number;
+};
+
+export type ACCOUNT_SOCIAL = {
+  id: string;
+  social: SOCIAL;
+  social_id: string;
+  account_id: string;
+};
+
+export type ACCOUNT_INFO = {
   account_id: string;
-  account_socials: Array<SOCIAL>;
-  created_at: string;
   id: string;
   image: string | null;
   instruction: string | null;
-  knowledges: any;
+  knowledges: Array<string>;
   label: string | null;
-  name: string | null;
-  organization: string | null;
+  organization: string;
+  updated_at: string;
+};
+
+export type ARTIST_INFO = {
+  account_id: string;
+  artist_id: string;
+  id: string;
   updated_at: string;
+  artist: {
+    id: string;
+    name: string;
+    timestamp: number | null;
+    account_socials: Array<ACCOUNT_SOCIAL>;
+    account_info: Array<ACCOUNT_INFO>;
+  };
```

Comment:
why nest this `artist` object?
- it seems to cause a lot of `artist.artist` blocks of code.
- what happens if you move all attributes in `artist` to the `root` object for easier accessibility?